Documentation/Testing/ProjectTesting.rst

```diff
@@ -6,262 +6,234 @@
-    }
-
-It just calls the homepage of our instance, clicks one of the links and verifies some text is
-shown. Straight, but enough to see if the basic instance does work.
-
-Ah, and we need a "Tester" in the `Support directory <https://github.com/TYPO3-Documentation/site-introduction/tree/main/Tests/Acceptance/Support>`_.
-
-That's it. We can now execute the acceptance test suite by executing a command in the
-ddev PHP container:
-
-.. code-block:: shell
-
-    lolli@apoc /var/www/local/site-introduction $ ddev exec bin/codecept run acceptance -d -c Tests/codeception.yml
-    Codeception PHP Testing Framework v2.5.6
-    Powered by PHPUnit 7.5.20 by Sebastian Bergmann and contributors.
-    Running with seed:
-
-
-    Bk2k\SiteIntroduction\Tests\Acceptance\Support.acceptance Tests (1) -------------------------------------------------------------------------------------------------
-    Modules: Asserts, WebDriver
-    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------
-    ⏺ Recording ⏺ step-by-step screenshots will be saved to /var/www/html/Tests/../var/log/_output/
-    Directory Format: record_5be441bbdc8ed_{filename}_{testname} ----
-    FrontendPagesCest: First page is rendered
-    Signature: Bk2k\SiteIntroduction\Tests\Acceptance\Frontend\FrontendPagesCest:firstPageIsRendered
-    Test: Acceptance/Frontend/FrontendPagesCest.php:firstPageIsRendered
-    Scenario --
-     I am on page "/"
-      [GET] https://introduction.ddev.site/
-     I see "Open source, enterprise CMS delivering  content-rich digital experiences on any channel,  any device, in any language"
-     I click "Customize"
-     I see "Incredible flexible"
-     PASSED
-
-    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------
-    ⏺ Records saved into: file:///var/www/html/Tests/../var/log/_output/records.html
-
-
-    Time: 8.46 seconds, Memory: 8.00MB
-
-    OK (1 test, 2 assertions)
-
-    lolli@apoc /var/www/local/site-introduction $
-
-Done: Local test execution of a projects acceptance test!
-
-.. index:: Testing; GitHub Actions
-
-GitHub Actions
-==============
-
-With local testing in place, we now want tests to run automatically when something is merged
-into the repository, and when people create pull requests for our project, we want to make sure
-that our carefully crafted test setup actually works. We're going to use Github's Actions CI
-service to get that done. It's free for open source projects.
-To tell the CI what to do, create a new workflow file in
-`.github/workflows/tests.yml <https://github.com/TYPO3-Documentation/site-introduction/blob/master/.github/workflows/tests.yml>`__
-
-.. code-block:: yaml
+The tests can be run via PHP on your local machine or with DDEV.
+
+.. _testing-projects-execution-cs:
+
+Run the PHP cs fixer
```

Comment:
In various parts of the doc you write "php-cs-fixer". You can keep it, but that writing is different than all other mentions of that package

Reply:
I think here we also need a unique spelling, I think "PHP-CS-Fixer" is the right way

Reply:
It is not in our Glossary yet: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/Glossary.html would one of you like to make a descision and make a PR there? then we can do search and replace all over.